doc/README

```diff
@@ -0,0 +1,14 @@
+===========================================
+NISTATS: functional MRI analysis in python
+===========================================
+
+This repository contains code and documentation to perform fMRI analysis. The requirements are ...
```

Comment:
I believe that your project is to merge that one and then proceed with content refinement, right ?